h/presenters/annotation_json.py

```diff
@@ -21,18 +18,7 @@ def __init__(self, annotation_resource, formatters=None):
         self._formatters = []
 
         if formatters is not None:
-            for formatter in formatters:
-                self._add_formatter(formatter)
-
-    def _add_formatter(self, formatter):
-        try:
-            verifyObject(IAnnotationFormatter, formatter)
-        except DoesNotImplement as err:
-            raise ValueError(
-                "formatter is not implementing IAnnotationFormatter interface"
-            ) from err
-
-        self._formatters.append(formatter)
+            self._formatters.extend(formatters)
```

Comment:
I don't think we need this level of defensive programming. If we are still keen we could get around this pretty easily with an `assert isinstance`

Reply:
Yeah we don't need that. I think if we wanted that sort of thing type-checking and actually using mypy might be the way to go?